src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -117,6 +127,13 @@ private void reportSLOs(final long startingNanos, final int totalSizeBytes, fina
 
             LOG.info("[SLO] [publishing-latency] time={} size={} count={} eventTypeName={} app={}", msSpent,
                     totalSizeBytes, eventCount, eventTypeName, applicationName);
+
+            nakadiKpiPublisher.publish(kpiBatchPublishedEventType, () -> new JSONObject()
```

Comment:
I understand that "msSpent" is not needed for your task, but do you think that we may need it in the future? 
Or maybe we can keep the event-type forward compatible and add it later if needed?

Reply:
I'll add right now, just because we can =)

Reply:
@v-stepanov done https://github.com/zalando/nakadi/pull/812/commits/a62648ee65310078e2ef236c4396288358d9d0cc